src/main/java/uk/gov/hmcts/reform/sscs/service/NotificationValidService.java

```diff
@@ -37,6 +38,15 @@ static boolean isFallbackLetterRequiredForSubscriptionType(NotificationWrapper w
         return result;
     }
 
+    static boolean fallbackConditionsMet(NotificationWrapper wrapper, NotificationEventType eventType) {
+        if (SYA_APPEAL_CREATED_NOTIFICATION.equals(eventType)) {
+            return (null == wrapper.getOldSscsCaseData() || wrapper.getOldSscsCaseData().getCaseReference().isEmpty())
+                && !wrapper.getNewSscsCaseData().getCaseReference().isEmpty();
+        }
+
+        return true;
```

Comment:
If the notification type is not `SYA_APPEAL_CREATED_NOTIFICATION` should the return not be false?